tests/transformers/test_transformers_model_export.py

```diff
@@ -422,6 +455,9 @@ def test_qa_model_save_and_override_card(small_qa_pipeline, model_path):
     # Validate inferred model card text
     with model_path.joinpath("model_card.md").open() as file:
         card_text = file.read()
+    # verify the license file has been written
+    license_file = model_path.joinpath("LICENSE.txt").read_text()
```

Comment:
```suggestion
    assert model_path.joinpath("LICENSE.txt").stat().st_size > 0
```

not a blocker

Reply:
changed to this for a number of reasons related to inconsistencies with the huggingface_hub APIs in older versions (reversions, renames, etc.)